.travis.yml

```diff
@@ -33,5 +33,5 @@ deploy:
     tags: true
 
 after_deploy:
-  - cd cmd/native-driver-performance/ ; CGO_ENABLED=0 go build -o ../../build/native-driver-performance ; cd ../..
+  - cd cmd/native-driver-performance/ ; CGO_ENABLED=0 go build -o ../../build/bin/native-driver-performance ; cd ../..
```

Comment:
if we have an extra dir `bin`, should not we add an extra level to cd (`cd ../../..`)?

Reply:
I guess no because initially we're on `cmd/native-driver-performance/`